android/src/main/java/com/reactcommunity/rndatetimepicker/CustomTimePickerDialog.java

```diff
@@ -0,0 +1,220 @@
+          // set valid minutes && move caret to the end of input
+          view.setCurrentMinute(snapMinutesToInterval(realMinutes));
+          view.setCurrentHour(hourOfDay);
+          textInput.setSelection(textInput.getText().length());
+        }
+      };
+
+      handler.postDelayed(runnable, 500);
+      return;
+    }
+
+    view.setCurrentMinute(snapMinutesToInterval(realMinutes));
+    view.setCurrentHour(hourOfDay);
+  }
+
+  @Override
+  public void onTimeChanged(final TimePicker view, final int hourOfDay, final int minute) {
+    final int realMinutes = getRealMinutes(minute);
+
+    // remove pending 'validation' callbacks if any
+    handler.removeCallbacks(runnable);
+
+    if (minutesAreInvalid(realMinutes)) {
+      correctEnteredMinutes(view, hourOfDay, realMinutes);
+      return;
+    }
+
+    super.onTimeChanged(view, hourOfDay, realMinutes);
+  }
+
+  @Override
+  public void onClick(DialogInterface dialog, int which) {
+    switch (which) {
+      case BUTTON_POSITIVE:
+        final int hours = mTimePicker.getCurrentHour();
+        final int realMinutes = getRealMinutes(mTimePicker.getCurrentMinute());
+        int minutesToCallListenerWith = realMinutes;
+
+        if (pickerIsInTextInputMode() && minutesAreInvalid(realMinutes)) {
+          minutesToCallListenerWith = snapMinutesToInterval(realMinutes);
+        }
+
+        if (mTimeSetListener != null) {
+          mTimeSetListener.onTimeSet(mTimePicker, hours, minutesToCallListenerWith);
+        }
+        break;
+      case BUTTON_NEGATIVE:
+        cancel();
+        break;
+    }
+  }
+
+  @Override
+  public void updateTime(int hourOfDay, int minuteOfHour) {
+    mTimePicker.setCurrentHour(hourOfDay);
+    mTimePicker.setCurrentMinute(snapMinutesToInterval(minuteOfHour));
+  }
+
+  /**
+   * Apply visual style in 'spinner' mode
+   * Adjust minutes to correspond selected interval
+   */
+  @Override
+  public void onAttachedToWindow() {
+    super.onAttachedToWindow();
+    int timePickerId = mContext.getResources().getIdentifier("timePicker", "id", "android");
+
+    try {
+      mTimePicker = this.findViewById(timePickerId);
+      int currentMinute = mTimePicker.getCurrentMinute();
+
+      if (mDisplay == RNTimePickerDisplay.SPINNER) {
+        int minutePickerId = mContext.getResources().getIdentifier("minute", "id", "android");
+        NumberPicker minutePicker = this.findViewById(minutePickerId);
+
+        minutePicker.setMinValue(0);
+        minutePicker.setMaxValue((60 / mTimePickerInterval) - 1);
+
+        List<String> displayedValues = new ArrayList<>();
+        for (int i = 0; i < 60; i += mTimePickerInterval) {
+          displayedValues.add(String.format(Locale.US, "%02d", i));
+        }
+
+        minutePicker.setDisplayedValues(displayedValues.toArray(new String[0]));
+      }
+
+      mTimePicker.setCurrentMinute(snapMinutesToInterval(currentMinute));
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
why try-catch? what can go wrong here?

Reply:
Removed this try catch, honestly I don't know what's problems can occurs in this method, I've tested without the catch with no problems